### PR TITLE
feat(components/molecule/collapsible): add new tokens for show more collapsible button

### PR DESCRIPTION
**Pull request by quinwacca**
## [molecule]/[collapsible]
**TASK**: [MTR-50180](https://jira.scmspain.com/browse/MTR-50180)

### Types of changes
Add new tokens to be modified on motor themes.

- [x] New feature (non-breaking change which adds functionality)

### CODE REVIEW

**Review by jordevo — APPROVED** [review]

**Review by nucliweb — APPROVED** [review]

File: components/molecule/collapsible/src/_settings.scss [hunk 1]
```diff
@@ -6,12 +6,14 @@ $bg-collapsible-container-gradient: linear-gradient(
   $bgc-collapsible-container-transparent 80%
 );
 $c-collapsible-btn: $c-primary !default;
+$fw-collapsible-btn: normal !default;
```
Comment (turolopezsanabria) [inline components/molecule/collapsible/src/_settings.scss]:
```suggestion
$fw-collapsible-btn: $fw-regular !default;
```

**Review by turolopezsanabria — COMMENTED** [review]
I suggest using $fw-regular to stick to the theme
thnx!

File: components/molecule/collapsible/src/_settings.scss (hunk 1 above)
Comment (quinwacca) [reply components/molecule/collapsible/src/_settings.scss]: Done! Thanks for the suggestion

**Review by turolopezsanabria — APPROVED** [review]
Thanks!

**Review by xmurcia — APPROVED** [review]

**Review by seremejvaz — APPROVED** [review]

**Review by andresin87 — APPROVED** [review]

**Pull request merged**